fsspec/implementations/zip.py

```diff
@@ -75,6 +82,16 @@ def _get_dirs(self):
                 )
                 self.dir_cache[f["name"]] = f
 
+    def info(self, path, **kwargs):
+        self._get_dirs()
+        path = self._strip_protocol(path)
+        if path in self.dir_cache:
+            return self.dir_cache[path]
+        elif path + "/" in self.dir_cache:
```

Comment:
These are directories?

Reply:
Yes. The `dir_cache` stores directories with `/` appended to them.